### PR TITLE
Support Kerberos authentication

### PR DESCRIPTION
**Pull request by falox**
Support Kerberos authentication
Signed-off-by: Alberto Falossi <afalossi@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Updates the Python 3.8 based custom image `Dockerfile` to support Kerberos authentication

## Description

This PR extends the `Dockerfile` to install the dependencies to access Red Hat services via Kerberos. The `krb5.conf` overrides the default one with the settings for the Red Hat Kerberos realm.

### CODE REVIEW

File: overlays/python38/Dockerfile [hunk 1]
```diff
@@ -11,6 +11,26 @@ ENV \
     # Path to be used in other layers to place s2i scripts into
     STI_SCRIPTS_PATH=/opt/app-root/builder
 
+USER root
```
Comment (harshad16) [inline overlays/python38/Dockerfile]: can you please copy the changes and create an new subdir:
`overlays/python38-internal`
and move this changes there

**Review by harshad16 — CHANGES_REQUESTED** [review]
Thanks for the pull request @falox,
this is great as this would need to run behind VPN.

File: overlays/python38/Dockerfile (hunk 1 above)
Comment (falox) [reply overlays/python38/Dockerfile]: @harshad16 moved!

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**